rmills/bin/publish

```diff
@@ -162,7 +162,7 @@ for ( var n = 0 ; n < names.length ; n++ ) {
 	var pname 	= n == 0 		? "default" : names[n-1] ;
 	var nname 	= n == (l-1)	? "default" : names[n+1] ;
 	var path    = dir + name + ".shtml" ;	
-	var defolt  = 'default'             ;
+	var default_  = 'default'             ;
```

Comment:
I could also drop this, but, if not, I should probably adjust the whitespace. Not sure what the intended alignment is.... perhaps `path` should align w/ `nname`?

Reply:
I've decided to leave this overhanging, it matches the previous section (but I've trimmed the space before `;`)